projects/kit/src/lib/masks/number/processors/fullwidth-to-halfwidth-preprocessor.ts

```diff
@@ -0,0 +1,20 @@
+import {MaskitoPreprocessor} from '@maskito/core';
+
+import {toHalfWidthNumber} from '../utils/to-half-width-number';
+
+/**
+ * Convert full width numbers like １, ２ to half width numbers 1, 2
+ */
+export function createFullWidthToHalfWidthPreprocessor(): MaskitoPreprocessor {
+    return ({elementState, data}) => {
+        const {value, selection} = elementState;
+
+        return {
+            elementState: {
+                selection,
+                value: toHalfWidthNumber(value),
+            },
+            data: toHalfWidthNumber(data),
```

Comment:
@rikusen0335 I have a small question to you again)

Should we use this same preprocessor for [`Time`](https://maskito.dev/kit/time), [`Date`](https://maskito.dev/kit/date), [`DateRange`](https://maskito.dev/kit/date-range), [`DateTime`](https://maskito.dev/kit/date-time) too ?

Are these built-in masks relevant for Japanese ?

Reply:
Ahhh probably yes, we can input full width numbers on them

Reply:
@rikusen0335 Thank you!

I've created an issue for it
* https://github.com/taiga-family/maskito/issues/884